tests/functional/aws-node-sdk/test/quota/tooling.js

```diff
@@ -33,18 +33,20 @@ const sendRequest = async (method, host, path, body = '', config = null) =>
             options.body = request.body;
         }
 
-        try {
-            const response = await fetch(url, options);
-            const text = await response.text();
-            const result = await xml2js.parseStringPromise(text);
-            if (result && result.Error) {
-                reject(result);
-            } else {
-                resolve(result);
+        (async () => {
```

Comment:
why do we need this?

since the code is already async/wait, we can simply remove the outer Promise, and keep the function async:

```
const sendRequest = async (method, host, path, body = '', config = null) =>
    const service = 's3';
    const endpoint = new AWS.Endpoint(host);
   
    [...]
   
    const response = await nodeFetch(url, options);
    const text = await response.text();
    const result = await xml2js.parseStringPromise(text);
    if (result && result.Error) {
        throw result;
    }

    return result;
}
```